scripts/submitArmadaSpark.sh

```diff
@@ -54,4 +57,8 @@ docker run -v $scripts/../conf:/opt/spark/conf --rm --network host $IMAGE_NAME \
     --conf spark.armada.container.image=$IMAGE_NAME \
     --conf spark.armada.jobSetId="$JOBSET" \
     --conf spark.armada.scheduling.nodeUniformity=armada-spark \
+    --conf spark.kubernetes.container.image=$IMAGE_NAME \
```

Comment:
Can we just use `spark.armada.container.image` and set this in code when needed?

Reply:
good idea, done.